test/GovernorBravo.test.ts

```diff
@@ -0,0 +1,253 @@
+import {solidity} from 'ethereum-waffle'
+import {SignerWithAddress} from '@nomiclabs/hardhat-ethers/signers'
+import {advanceBlockTo} from './utils/index'
+import {
+    GovernorAlpha,
+    GovernorBravoDelegate,
+    GovernorBravoDelegator,
+    BitDAO,
+    Treasury,
+    Timelock
+} from '../typechain'
+
+// Wires up Waffle with Chai
+chai.use(solidity)
+
+const TREASURY_FUNDS = '1000000000000000000000000000'
+const VOTING_POWER = '100000000000000000000000000'
+
+const provider = ethers.provider
+const PROPOSAL_SPAN = 5
+const abiCoder = new ethers.utils.AbiCoder()
+// Governance.ProposalState in contracts/Governance.sol.
+enum ProposalState {
+    Pending,
+    Active,
+    Canceled,
+    Defeated,
+    Succeeded,
+    Queued,
+    Expired,
+    Executed
+}
+
+describe('GovernorBravoDelegate', function () {
+    before(async function () {
+        this.signers = await ethers.getSigners()
+        this.admin = this.signers[0]
+        this.delegatee1 = this.signers[1]
+        this.delegatee2 = this.signers[2]
+        this.delegatee3 = this.signers[3]
+        this.receiver = this.signers[4]
+        this.GovernorAlpha = await ethers.getContractFactory('GovernorAlphaV8')
+        this.GovernorBravoDelegate = await ethers.getContractFactory(
+            'GovernorBravoDelegate'
+        )
+        this.GovernorBravoDelegator = await ethers.getContractFactory(
+            'GovernorBravoDelegator'
+        )
+        this.Timelock = await ethers.getContractFactory('TimelockV8')
+        this.BitDAO = await ethers.getContractFactory('BitDAO')
+        this.Treasury = await ethers.getContractFactory('Treasury')
+        this.timelockAlpha = <Timelock>(
+            await this.Timelock.deploy(this.admin.address, 1)
+        )
+        await this.timelockAlpha.deployed()
+        this.timelock = <Timelock>(
+            await this.Timelock.deploy(this.admin.address, 1)
+        )
+        await this.timelock.deployed()
+        this.bit = <BitDAO>await this.BitDAO.deploy(this.admin.address)
+        await this.bit.deployed()
+        this.governorAlpha = <GovernorAlpha>(
+            await this.GovernorAlpha.deploy(
+                this.timelockAlpha.address,
+                this.bit.address,
+                this.admin.address
+            )
+        )
+        await this.governorAlpha.deployed()
+        this.governorDelegate = <GovernorBravoDelegate>(
+            await this.GovernorBravoDelegate.deploy()
+        )
+        await this.governorDelegate.deployed()
+        this.governorDelegator = <GovernorBravoDelegator>(
+            await this.GovernorBravoDelegator.deploy(
+                this.timelock.address,
+                this.bit.address,
+                this.admin.address,
+                this.governorDelegate.address,
+                PROPOSAL_SPAN,
+                0,
+                VOTING_POWER
+            )
+        )
+        await this.governorDelegator.deployed()
+        // Attach implementation.
```

Comment:
question: what does this comment mean?

The comment either needs expanding on or deleting.